scripts/install.sh

```diff
@@ -210,6 +215,14 @@ install_docker() {
         echo
         sudo yum install docker
         sudo service docker start
+    elif [[ $package_manager == pacman ]]; then
+        echo "Installing docker"
+        # sudo pacman -Syu
+        sudo pacman -S docker --noconfirm # (docker, containerd, runc)
+        sudo pacman -S docker-compose --noconfirm
```

Comment:
I suppose docker-compose should be installed in the `install_docker_compose` block in case the user has docker installed and not docker compose